packages/grafana-data/src/transformations/transformers/labelsToFields.ts

```diff
@@ -64,6 +65,46 @@ export const labelsToFieldsTransformer: DataTransformerInfo<LabelsToFieldsOption
   },
 };
 
+function getFramesWithOnlyValueFields(data: DataFrame[]): DataFrame[] {
+  const processed: DataFrame[] = [];
+
+  for (const series of data) {
+    const fields: Field[] = [];
+
+    for (let i = 0; i < series.fields.length; i++) {
+      const field = series.fields[i];
+      if (field.type !== FieldType.number) {
```

Comment:
what about the time field?  Is that not wanted here?

Reply:
not in this function, for some reason, the data is split into data frames with time fields and the data frames without, not sure why.

Reply:
yes, not splitting the frames like I did is one optimization we could do if we encounter perf issues with current solution.